tests/plugins/wysiwygarea/manual/detachobserver.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: 4.17.2, feature, 4462
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea,toolbar,basicstyles,link,format,sourcearea,elementspath,undo
+
+1. Open dev console to track possible errors.
+2. Wait until the `Trigger test` button will be available.
+3. Click the button and watch console.
+
+	**Expected:** No errors in the console.
+
+	**Unexpected:** Error appears: `Uncaught ReferenceError: CKEDITOR is not defined`.
```

Comment:
```suggestion
**Expected:** No errors in the console.

**Unexpected:** Error appears: `Uncaught ReferenceError: CKEDITOR is not defined`.
```
This additional blank space is not correct with our [code style](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#31-manual-tests). Please adjust it.